conf/messages.en

```diff
@@ -211,7 +211,6 @@ cf.search.authorities.result.company.name = Company name
 cf.search.authorities.result.account.balance.available = Balance available
 cf.search.authorities.result.balance.title = Why can I not see an account balance?
 cf.search.authorities.result.balance.label = The company that owns the account can choose to keep the balance private. You can still use the account.
-cf.search.authorities.result.name = The business this EORI number is registered to has not consented to their name being shared.
```

Comment:
This isn't used and is a duplicate of cf.search.authorities.result.insert. 
I'm just tidying it up.